packages/react-app/src/components/history/HistoryItem.jsx

```diff
@@ -310,29 +322,26 @@ export const HistoryItem = ({
           </Text>
           <AddToMetamask token={toToken} ml="0.25rem" />
         </Flex>
-        {receivingTx ? (
+        {/* {receivingTx ? (
```

Comment:
Could you tell me why you have removed this code? We need to show `Claimed` in case its executed.